eclass/vdr-plugin-2.eclass

```diff
@@ -97,10 +95,11 @@ DESCRIPTION="vdr Plugin: ${VDRPLUGIN} (based on vdr-plugin-2.eclass)"
 S="${WORKDIR}/${VDRPLUGIN}-${PV}"
 
 # depend on headers for DVB-driver and vdr-scripts
-DEPEND=">=media-tv/gentoo-vdr-scripts-0.4.2
+BDEPEND="dev-util/pkgconfig"
```

Comment:
This should be conditional. Add to `DEPEND` in EAPIs 5 and 6, `BDEPEND` in newer EAPIs.

Reply:
Also, why is this not virtual/pkgconfig?

Reply:
Yes, learned that from repoman. Done.

Reply:
Hello @ionenwks ,
the change you requested is solved, would you mark you review as complete like @ulm ?

Reply:
It was just a quick comment rather than a review (github is just annoying with this), haven't really looked at the eclass, I'm sure it's fine but I don't want to approve of something I haven't looked at.